content/learn/useCases/maintenanceScheduling.adoc

```diff
@@ -5,6 +5,8 @@
 :jbake-related_tag: maintenance scheduling
 :showtitle:
 
+// TODO show use case video NPTzXIyQdqU ?
```

Comment:
Wrong ID (there was the call center video)
```suggestion
// TODO show use case video gIaHtATz6n8 ?
```

Reply:
Thanks for catching this :)